### PR TITLE
Logout and quit prompts and other small stuff.

### PR DESCRIPTION
**Pull request by zambonin**
According to #153.

### CODE REVIEW

File: rtv/content.py [hunk 1]
```diff
@@ -131,9 +131,12 @@ def strip_praw_submission(sub):
         data['flair'] = flair
         data['url_full'] = sub.url
 
+        if flair is not None and flair[0] != '[':
```
Comment (michael-lazar) [inline rtv/content.py]: This will crash if the flair is an empty string, I would suggest using flair.startswith() instead.

Do some of the flairs already have brackets added? I haven't seen any.

File: rtv/content.py [hunk 2]
```diff
@@ -131,9 +131,12 @@ def strip_praw_submission(sub):
...
+            data['flair'] = '[{}]'.format(flair.strip())
+
         if data['permalink'].split('/r/')[-1] == data['url_full'].split('/r/')[-1]:
             data['url_type'] = 'selfpost'
-            data['url'] = 'selfpost'
+            data['url'] = 'self.{}'.format(data['subreddit'])
```
Comment (michael-lazar) [inline rtv/content.py]: Might as well do the same thing for x-posts to maintain a consistent style.

File: rtv/oauth.py [hunk 3]
```diff
@@ -106,7 +106,7 @@ def authorize(self):
         except (praw.errors.OAuthAppRequired, praw.errors.OAuthInvalidToken):
             show_notification(self.stdscr, ['Invalid OAuth data'])
         else:
-            message = ['Welcome {}!'.format(self.reddit.user.name)]
+            message = ['Welcome, {}!'.format(self.reddit.user.name)]
```
Comment (michael-lazar) [inline rtv/oauth.py]: Unnecessary change

File: rtv/content.py (hunk 1 above)
Comment (zambonin) [inline rtv/content.py]: All flairs from /r/bestof have tags. Will fix.

File: rtv/content.py (hunk 2 above)
Comment (zambonin) [inline rtv/content.py]: If I understood correctly what you meant, the last change addresses this.

**Comment by zambonin** [comment]
Should be all fixed. :)

**Comment by michael-lazar** [comment]
Merged, thanks again for the PR!

**Comment by shaggytwodope** [comment]
@michael-lazar @zambonin I understand this pr and such been merged already. But would it be worth considering a config option to bypass this behavior? As I rather enjoyed the previous behavior.

**Comment by michael-lazar** [comment]
@shaggytwodope. I do like the idea of having a confirmation dialog. Skipping the *Enter* and quitting right after `qy` would be an easy way to bring the keystrokes down from 3 to 2. Or perhaps you could use capital `Q` to force quit without the confirmation. Do you think either of these options would be sufficient?

**Comment by shaggytwodope** [comment]
@michael-lazar  Oh indeed a capital q would would be the best of both worlds I think.

**Comment by zambonin** [comment]
@shaggytwodope @michael-lazar It's implemented. I'm currently trying to solve another issue and will make the PR soon.
